code/modules/mob/living/simple_animal/abnormality/aleph/staining_rose.dm

```diff
@@ -30,43 +32,55 @@
 	gift_type = /datum/ego_gifts/blossoming
 
 	var/chosen
-	var/meltdown_cooldown_time = 300 SECONDS
+	var/list/sacrificed = list()
+	var/list/heretics = list()
+	var/meltdown_cooldown_time = 15 MINUTES
 	var/meltdown_cooldown
+	var/safe = FALSE //work on it and you're safe for 15 minutes
+	var/reset_time = 3 MINUTES //Qliphoth resets after this time
 
 
+/mob/living/simple_animal/hostile/abnormality/staining_rose/Initialize()
+	..()
+	meltdown_cooldown = world.time + meltdown_cooldown_time
+
 /mob/living/simple_animal/hostile/abnormality/staining_rose/work_complete(mob/living/carbon/human/user, work_type, pe)
 	..()
-	datum_reference.qliphoth_change(1)
+	safe = TRUE
 	if (chosen == null)
 		chosen = user
 		user.visible_message("<span class='warning'>You are now the rose's chosen.</span>")
 		icon_state = "rose_activated"
 
 	if (user != chosen)		//Your body starts to wilt.
 		user.visible_message("<span class='warning'>The rose has already chosen another, [chosen]!</span>")
-		user.physiology.red_mod *= 1.2
-		user.physiology.white_mod *= 1.2
-		user.physiology.black_mod *= 1.2
-		user.physiology.pale_mod *= 1.2
+		user.apply_status_effect(STATUS_EFFECT_SCHISMATIC)
+		if(!(user in heretics))
+			heretics += user
 		pissed()
 
 	else
 		user.visible_message("<span class='warning'>The Rose is satisfied.</span>")
 
 	if(get_attribute_level(user, JUSTICE_ATTRIBUTE) < 100)
-		user.visible_message("<span class='warning'>The Rose is sapping your strength</span>")
-		user.physiology.red_mod *= 1.1
-		user.physiology.white_mod *= 1.1
-		user.physiology.black_mod *= 1.1
-		user.physiology.pale_mod *= 1.1
-		datum_reference.qliphoth_change(1)	//If you lose your defenses you increase it's Q counter again
+		user.apply_status_effect(STATUS_EFFECT_SACRIFICE)
+		if(!(user in sacrificed))
+			sacrificed += user
+			user.visible_message("<span class='warning'>The rose adds your strength to it, and it is born anew.</span>")
+			meltdown_cooldown = world.time + meltdown_cooldown_time	//There we go!
 		pissed()
 
 /mob/living/simple_animal/hostile/abnormality/staining_rose/Life()
 	. = ..()
-	if(meltdown_cooldown < world.time && !datum_reference.working)
+	if(meltdown_cooldown < world.time)
```

Comment:
Why remove the working check? This means it can meltdown during works again

Reply:
That was a bug with my github